src/features/governance/hooks/useGovernanceProposals.ts

```diff
@@ -157,8 +157,12 @@ export function useGovernanceProposals() {
       const votes = votesResults.votes?.[proposal.id!];
       computeStageAndVotes(proposal, votes);
     });
-
-    return proposals_?.sort((a, b) => b.metadata!.cgp - a.metadata!.cgp);
+    return proposals_?.sort((a, b) => {
+      if (!a.id && !b.id) {
```

Comment:
why not a `||` so that if either is missing compare by cgp rather than 1 be missing and we compare by nothing.

Reply:
so having a proposalId always has priority, but this is irrelevant now that drafts only appear in the draft tab

Reply:
ok